default.json

```diff
@@ -53,6 +55,7 @@
             "default_backend_title": "wire-staging",
             "encrypt_proteus_storage": true,
             "analytics_enabled": true,
+            "picture_in_picture_enabled": true,
```

Comment:
request: Staging and beta are supposed to mimic prod features, can you please disable it there